kin-recovery/README.md

```diff
@@ -0,0 +1,180 @@
+# Backup and Restore Standalone Module
+
+This module is an optional way to back up and/or restore your account.  
+The module wraps the Kin SDK Android import and export functionalities with a UI that includes two flows - backup and restore.  
+The UI uses a password to create a QR code, which is then used to back up the account and to restore it.
+
+It is implemented as an Android library that can be incorporated into your code.  
+This library is dependent on the kin-sdk library, and we assume that whoever needs to use it is already familiar with the kin-sdk library.
+For more details on Kin SDK, go to [kin-sdk](https://github.com/kinecosystem/kin-sdk-android/tree/master/kin-sdk).
+
+## Installation
+
+The Kin Recovery module for Android is implemented as an Android library.  
+To include the library in your project, add these two statements to your `build.gradle` files.
+
+###### Snippet: Modifying project build file
+
+```gradle
+allprojects {
+    repositories {
+...
+        maven {
+            url 'https://jitpack.io'
+        }
+    }
+}
+```
+###### Snippet: Modifying module build files
+
+```gradle
+...
+dependencies {
+    ...
+    implementation 'com.github.kinecosystem.kin-sdk-android:recovery:<latest release>'
+}
+```
+
+For the latest release version, go to [https://github.com/kinecosystem/kin-sdk-android/releases](https://github.com/kinecosystem/kin-sdk-android/releases).
+
+See the main repository at [github.com/kinecosystem/kin-sdk-android](https://github.com/kinecosystem/kin-sdk-android).
+
+
+## Overview
+
+Launching Backup and Restore flows requires the following steps:
+
+1. Creating the backup and restore manager
+2. Adding callbacks
+3. Passing the result
+4. Backup or Restore
+
+### Step 1 - Creating the Backup and Restore Manager
+
+First you will need to have a KinClient object for later use when you will invoke backup and/or restore methods.  
+Also you need to create the BackupAndRestoreManager object.
+###### Snippet: This is an example of how to create those objects:
+
+```java
+kinClient = new KinClient(context, Environment.TEST, "1acd")
+...
+backupAndRestoreManager = new BackupAndRestoreManager(context);
+```
+
+
+For more details on KinClient, see https://github.com/kinecosystem/kin-sdk-android/tree/master/kin-sdk.
```

Comment:
probably needs a more accurate link to docs /readme

Reply:
I am not sure how to create a link which goes exactly to a specific section in the readme.
Do you know how?

Reply:
sure, you can link any anchor like this:
https://github.com/kinecosystem/kin-sdk-android#accessing-the-kin-blockchain
(that's for instance where we start explaining on `KinClient`)
in any case, the problem with the link you provided, is that it's points to a folder which doesn't give actual more details to the reader